### PR TITLE
Chage 'Rotten' button

### PR DESCRIPTION
**Pull request by agravem**
More info here: https://www.pivotaltracker.com/story/show/121752785

### CODE REVIEW

**Comment by agravem** [comment]
@AhmedBelal @maletor @DanielNill please review

File: spec/models/update_request_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe UpdateRequest, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (rstawarz) [inline spec/models/update_request_spec.rb]: If we're not adding any specs, we should remove this file

**Comment by AhmedBelal** [comment]
Your link doesn't point to any story, just a project.

File: spec/factories/update_requests.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+FactoryGirl.define do
+  factory :update_request do
+    article_id 1
+description "MyText"
```
Comment (rstawarz) [inline spec/factories/update_requests.rb]: indention.  I'd recommend instilling the [`precommit](https://github.com/jish/pre-commit)` gem to have it run rubucop and do some cursory checks on the code before committing

Comment (brupm) [inline spec/factories/update_requests.rb]: https://github.com/bbatsov/ruby-style-guide

File: app/views/update_requests/new.html.haml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+%section.row
+  .cell.well.well--l
+    .g
+      .g-b.g-b--m--2of3
```
Comment (AhmedBelal) [inline app/views/update_requests/new.html.haml]: gnarly naming

**Comment by AhmedBelal** [comment]
LGTM 👍

**Comment by DanielNill** [comment]
Not sure what other changes we've made to orientation, but it might be nice to bear in mind any future plans we might have to merge back in to the parent.

Otherwise code looks good 👍

File: app/views/update_requests/new.html.haml (hunk 3 above)
Comment (agravem) [inline app/views/update_requests/new.html.haml]: Yep. 😞

**Comment by brupm** [comment]
@DanielNill we do and @agravem is aware: https://github.com/doximity/orientation/blob/pseudo-master/CONTRIBUTING.md

**Pull request merged**